Calendar/CalLogHelpers/CalLogCSVFunctions.ps1

```diff
@@ -80,8 +72,12 @@ function CreateExternalMasterIDMap {
         $AllFolderNames = @($script:GCDO | Where-Object { $_.ExternalSharingMasterId -eq $ExternalID } | Select-Object -ExpandProperty OriginalParentDisplayName | Select-Object -Unique)
 
         if ($AllFolderNames.count -gt 1) {
-            # We have 2+ FolderNames, Need to find the best one. #remove Calendar
-            $AllFolderNames = $AllFolderNames | Where-Object { $_ -notmatch 'Calendar' } # This will not work for non-english
+            # We have 2+ FolderNames, Need to find the best one. Remove 'Calendar' from possible names
+            $AllFolderNames = $AllFolderNames | Where-Object {
+                $_ -notmatch 'Calendar' -and
+                $_ -notmatch 'Calendrier' -and
+                $_ -notmatch 'Kalender'
+            } # Need a better way to do this for other languages...
```

Comment:
I am thinking we should be able to line up between `Get-CalendarDiagnosticObjects` output and `Get-MailboxFolderStatistics -Identity NAME -FolderScope Calendar` on the name. That way it should catch all languages without any issues.

Reply:
I have done this before.  I will revert to English only until we can get a complete solution.